wiki/content/enterprise-features/index.md

```diff
@@ -273,6 +273,10 @@ backups built on top of it. Each time a new full backup is created, a new backup
 series with a different ID is started. The backup series ID is stored in each
 `manifest.json` file stored in every backup folder.
 
+The `--encryption_key_file` argument is required if you took the backup in an
```

Comment:
replace word "argument" with "flag" so its consistent with the rest of the section.